src/main/java/org/terasology/climateConditions/BodyTemperatureSystem.java

```diff
@@ -0,0 +1,124 @@
+// Copyright 2020 The Terasology Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+package org.terasology.climateConditions;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.terasology.entitySystem.entity.EntityManager;
+import org.terasology.entitySystem.entity.EntityRef;
+import org.terasology.entitySystem.event.ReceiveEvent;
+import org.terasology.entitySystem.systems.BaseComponentSystem;
+import org.terasology.entitySystem.systems.RegisterMode;
+import org.terasology.entitySystem.systems.RegisterSystem;
+import org.terasology.logic.characters.AliveCharacterComponent;
+import org.terasology.logic.chat.ChatMessageEvent;
+import org.terasology.logic.delay.DelayManager;
+import org.terasology.logic.delay.PeriodicActionTriggeredEvent;
+import org.terasology.logic.location.LocationComponent;
+import org.terasology.registry.In;
+import org.terasology.world.WorldComponent;
+
+
+@RegisterSystem(value = RegisterMode.AUTHORITY)
+public class BodyTemperatureSystem extends BaseComponentSystem {
+    public static final String BODY_TEMPERATURE_UPDATE_ACTION_ID = "Body Temperature Update";
```

Comment:
I don't think this makes a problem, but personally I cringe when I see identifiers with spaces 😅

Reply:
Ohh I thought it's a convention to name static final variables with all caps and then I noticed a lot of the Periodic Action Ids were named in a similar fashion ("....._UPDATE_ACTION_ID ") 😅. So would you prefer If I just remove the action id part or should I convert it to pascal case?

Reply:
Oh, I was talking about `"Body Temperature Update"` and not `BODY_TEMPERATURE_UPDATE_ACTION_ID` 😅 Out of habit I would try to make the action ID as unique as possible (probably using a similar pattern as for our resource URNs) 
```suggestion
    public static final String BODY_TEMPERATURE_UPDATE_ACTION_ID = "climateConditions:bodyTemperatureUpdate";
```

To the extreme, you could use a random UUID and the only way to reference the action is via this constant
```suggestion
    public static final String BODY_TEMPERATURE_UPDATE_ACTION_ID = "1bd16c1b-c662-4aae-8683-1467bc850ce4";
```

However, I'm not sure whether the id is printed out in some error cases, and using an UUID here would make it way harder to track down the problem...

In any case, this can be seen as an implementation detail that can be changed anytime.